src/plugins/intel_cpu/src/nodes/executors/acl/acl_convert.cpp

```diff
@@ -92,6 +92,12 @@ bool ACLConvertExecutorBuilder::isSupported(const ConvertParams& convertParams,
             DEBUG_LOG("NECopy does not support source precision: ", convertParams.srcPrc.to_string());
             return false;
         }
+        auto srcDataLayout = getAclDataLayoutByMemoryDesc(srcDesc);
+        auto dstDataLayout = getAclDataLayoutByMemoryDesc(dstDesc);
+        if (srcDataLayout == DataLayout::UNKNOWN || dstDataLayout == DataLayout::UNKNOWN) {
```

Comment:
@liubo-intel  So ARM issue started to reproduce after your changes?
I believe changes inside Deconv node should be applied for Intel CPUs only (should be done under x64 macro), because fp16 Deconvolution works well on ARM.